playbooks/preview.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
This is pretty cool, actually. Comprehensive coverage requires possibility of other override settings (via extraValueFiles, sharedValueFiles, and even potentially the possibility that clustergroup could be defined in values-global and something show up there. This does a great job for the stuff it does.

Another thing to think about adding at some point is a mechanism to add local files for consideration (which used to be possible via HELM_OPTS in the old scheme). This might actually be easier to add/do than the other one. I'm not sure how much that was actually used beyond That One Customer, though.